main/query_changes/permissions.py

```diff
@@ -90,6 +91,9 @@ def resources_queryset_for_user(
             .prefetch_related("creator")
         )
 
+    if not include_drafts:
+        init_queryset = init_queryset.filter(~Q(state="draft"))
```

Comment:
```suggestion
        init_queryset = init_queryset.exclude(state="draft")
```
Cette écriture me parait plus simple